internal/config/config_test.go

```diff
@@ -140,6 +142,15 @@ func TestReadTCConfig(t *testing.T) {
 		assert.Equal(t, expected, config)
 	})
 
+	t.Run("Invalid IP address", func(t *testing.T) {
```

Comment:
suggestion: to my previous set of comments, were working towards simple names with no spaces:
```suggestion
	t.Run("invalid-address", func(t *testing.T) {
```